nginx/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# NOTE: this dockerfile is primarilty for local development only
+#       it's primary purpose is to emulate heroku-buildpack-nginx's
+#       functionality that compiles config/nginx.conf.erb
+# See https://github.com/heroku/heroku-buildpack-nginx/blob/fefac6c569f28182b3459cb8e34b8ccafc403fde/bin/start-nginx
+FROM nginx:1.27.0
+
+# Logs are configured to a relatic path under /etc/nginx
+# but the container expects /var/log
+RUN mkdir -p /etc/nginx/logs && ln -sf /var/log/nginx /etc/nginx/logs/
+
+# erb unfortunately needs a whole ruby install
+RUN apt-get update && apt-get install -y ruby
+
+# this gets run automatically by the nginx container's entrypoint
+COPY 20-compile-nginx-conf-erb.sh /docker-entrypoint.d
```

Comment:
This script will not be executable inside the container once it's copied over. A line needs to be added after this to make it so:

`RUN chmod +x /docker-entrypoint.d/20-compile-nginx-conf-erb.sh`